posawesome/public/js/posapp/components/pos/Customer.vue

```
@@ -11,11 +10,13 @@
       bg-color="white"
       :disabled="readonly"
       append-inner-icon="mdi-arrow-right"
-      @click:append-inner="new_customer" 
+      @click:append-inner="new_customer"
+      @keyup.enter="new_customer"
       @click:prepend-inner="go_back"
+      @input="customer = customer.toUpperCase()"
```

ESLint

[error] VTextField: @input has been replaced with @update:modelValue

(vuetify/no-deprecated-events)